src/ElectricPokemon.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private String specialAttack;
+    private String strongAgainst;
+
+    public ElectricPokemon(String name, int hp, int level, String food, String sound) {
+        super(name, hp, level, food ,sound, "electric");
```

Comment:
"electric" is heir een "magische string" . JE tovert uit het niets deze string tevoorschijn zonder dat ik weet wat het moet voorstellen. Als je dit later wilt aanpassen, dan moet je ook je code doorzoeken waar je overal deze string hebt getoverd. 

Beter kun je boven aan je klasse een `private static final String TYPE = "electric"` zetten. Dan weet ik dat het een constante is die het type moet voorstlelen. Je kunt het dan vervolgens hier aanroepen en eventueel op andere plekken in je code. Als je dan later beslist dat het toch "volt" moet zijn, dan hoef je dat alleen in de constante aan te passen.